packages/web3-eth/test/integration/web3_eth/sign_transaction.test.ts

```diff
@@ -47,17 +48,21 @@ describe('Web3Eth.signTransaction', () => {
 			gasPrice: '0x3b9aca01',
 		};
 		const response = await web3Eth.signTransaction(transaction);
-		expect(response).toMatchObject({
+		const expectedResponse: { tx: Transaction } = {
 			tx: {
 				type: BigInt(0),
 				nonce: BigInt(nonce),
 				gasPrice: BigInt(1000000001),
 				gas: BigInt(21000),
 				value: BigInt(1),
 				to: transaction.to,
-				input: '0x',
 			},
-		});
+		};
+		if (getSystemTestBackend() === 'geth') expectedResponse.tx.input = '0x';
+		else if (getSystemTestBackend() === 'ganache') expectedResponse.tx.data = '0x';
```

Comment:
This indicates a breaking change.

We cannot be sure that no user used to fill `input` and then read `data` or vice versa.